src/Components/Popover/Base/Popover.scss

```diff
@@ -122,7 +122,7 @@
 
   strong {
     color: $g18-cloud;
-    font-weight: 900;
+    font-weight: 500;
```

Comment:
I'm a little confused about this change. In my mind, the `strong` tag is synonymous with `bold`. Rather than overwriting the default to normalize the font-weight, wouldn't we just not want to use `strong`?